core/integration/module_typescript_test.go

```diff
@@ -926,3 +926,61 @@ class Alias {
 		require.JSONEq(t, `{"alias": {"bar": {"hello": "a", "foo": {"zoo": 4, "hey": [true, false, true], "far": {"farFarNested": true} }}}}`, out)
 	})
 }
+
+func TestModuleTypeScriptPrototype(t *testing.T) {
+	t.Parallel()
+
+	t.Run("keep class prototype inside module", func(t *testing.T) {
+		t.Parallel()
+
+		c, ctx := connect(t)
+
+		modGen := c.Container().From(golangImage).
+			WithMountedFile(testCLIBinPath, daggerCliFile(t, c)).
+			WithWorkdir("/work").
+			With(daggerExec("init", "--name=test", "--sdk=typescript")).
+			With(sdkSource("typescript", `
+import { field, func, object } from "@dagger.io/dagger"
+
+@object()
+class Test {
+  @func() 
+	test() {
```

Comment:
There's several lines in this code that aren't indented correctly. Can you fix the formatting to make it easier to read?

Reply:
sure! Thanks for the catch!